data-prepper-plugins/translate-processor/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
     testImplementation project(':data-prepper-plugins:log-generator-source')
     testImplementation project(':data-prepper-test-common')
     implementation 'org.apache.commons:commons-lang3:3.12.0'
+    testImplementation project(path: ':data-prepper-plugins:mutate-event-processors')
```

Comment:
Why is this `testImplementation`?

Reply:
Wrong dependency. I with push changes with `implementation project(path: ':data-prepper-plugins:mutate-event-processors')`.